contracts/external/actions/VaultAccountAction.sol

```diff
@@ -268,6 +268,9 @@ contract VaultAccountAction is ActionGuards, IVaultAccountAction {
         bool transferSharesToLiquidator,
         bytes calldata redeemData
     ) external nonReentrant override returns (uint256 profitFromLiquidation) {
+        // Do not allow invalid accounts to liquidate
+        requireValidAccount(liquidator);
```

Comment:
Should you also require that the liquidator is not the vault itself as in the prior issue? I guess that's pretty unlikely, though you could just add require(msg.sender != vault) here: https://github.com/notional-finance/contracts-v2/blob/1f78fef0dfec70478b7ea00f40603e807c6d6af9/contracts/external/actions/VaultAccountAction.sol#L361

Reply:
Sure, we can do that